modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -0,0 +1,114 @@
+| `i3s.isTileset`    | Bool | `false` | Whether to load `Tileset` (Layer 3D Index) file. If not specifies, will decide if follow `argis` url convention |
+| `i3s.isTileHeader` | Bool | `false` | Whether to load `TileHeader`(node) file. If not specifies, will decide if follow `argis` url convention         |
+| `i3s.loadContent`  | Bool | `false` | Whether to load tile content (geometries, texture, etc.)                                                        |
+
+## Data formats
+
+This section specifies the loaded data formats.
+
+### Tileset Object
+
+The following fields are guaranteed. Additionally, the loaded tileset object will contain all the data fetched from the provided url.
+
+| Field            | Type     | Contents                                                                                                                                                                                                                                                                         |
+| ---------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `loader`         | `Object` | I3SLoader                                                                                                                                                                                                                                                                        |
+| `root`           | `Object` | The root tile header object                                                                                                                                                                                                                                                      |
+| `url`            | `String` | The url of this tileset                                                                                                                                                                                                                                                          |
+| `type`           | `String` | Value is `i3s`. Indicates the returned object is an `i3s` tileset.                                                                                                                                                                                                               |
+| `lodMetricType`  | `String` | Root's level of detail (LoD) metric type, which is used to decide if a tile is sufficient for current viewport. Only support `maxScreenThreshold` for now. Check I3S [lodSelection](https://github.com/Esri/i3s-spec/blob/master/docs/1.7/lodSelection.cmn.md) for more details. |
```

Comment:
Are these i3s specific? Maybe use a prefix or subobject?

Reply:
`lodMetricType` is `geometricError`(In this PR https://github.com/uber-web/loaders.gl/pull/654/files, mentioned in `modules/3d-tiles/docs/api-reference/tiles-3d-loader.md`) for `3d tiles`, as both of them are used to decide if tiles should be selected for current viewport. I am actually not sure if this makes sense. 

So maybe keep their original names with the prefix, `i3s.lodSelection`, `3d-tiles.geometricError`